package.json

```diff
@@ -12,8 +12,8 @@
     "pretest": "npm run lint && npm run prettier:check",
     "test": "jest",
     "test:update": "jest --updateSnapshot",
-    "deploy:production": "git push production master",
-    "deploy:staging": "git push staging master",
+    "deploy:production": "git push production main",
+    "deploy:staging": "git push staging main",
```

Comment:
These two commands are probably outdated, since instructions indicate that one should use `lerna` to deploy. @znarf can you confirm?

```suggestion
```